server/helpers/utilities.ts

```diff
@@ -8,8 +8,8 @@ function getStartMonth() {
   return format(startOfMonth(new Date()), 'yyyy-MM-dd')
 }
 
-const sortByDate = (data: CalendarDay[], dateField = 'date') =>
```

Comment:
tslint issue - second parameter was never used and always `date` so could just use the field instead